ERCS/erc-7695.md

```diff
@@ -0,0 +1,477 @@
+---
+eip: 7695
+title: Ownership Delegation and Context for ERC-721
+description: Extends ERC-721 for financial use cases without changing owner
+author: Duc Tho Tran (@ducthotran2010)
+discussions-to: https://ethereum-magicians.org/t/erc-7695-ownership-delegation-and-context-for-non-fungible-token/19716
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-04-02
+requires: 165, 721
+---
+
+## **Abstract**
+
+This proposal introduces an extension for [ERC-721](./eip-721.md) Non-Fungible Tokens (NFTs): specifying users for several contexts and differentiating owner and ownership rights for a token to use in several financial use cases for NFT without changing owner.
```

Comment:
Your abstract is pretty good, except maybe a tad too short. What differentiates your proposal from other similar ones? How does it work on a technical level?

Reply:
It was updated to:
_"This standard is an extension for ERC-721, designed to specify users for various contexts with a locking feature and allow temporary ownership delegation without changing the original owner._

_This EIP preserves the benefits and rights of the owner while expanding the utility of NFTs across various dApps by adding the concepts of Ownership Delegation and Contexts, which define specific roles: Controller and User, who can use the NFT within defined contexts."_ in this commit https://github.com/ethereum/ERCs/commit/064e1e7b60cb8275546c5c2649271a3a7428a9dc
Let me know if I can make it better. Thank you.